html/semantics/interactive-elements/the-dialog-element/dialog-show-modal.html

```diff
@@ -10,26 +10,29 @@
 
 <script>
 test(() => {
-  const dialog = document.getElementById('mydialog');
+  const dialog = document.getElementById("mydialog");
   const computedStyle = window.getComputedStyle(dialog, null);
-  assert_equals(computedStyle.display, 'none');
+  assert_equals(computedStyle.display, "none");
+  assert_false(dialog.matches(":modal"));
 
```

Comment:
Has this (the `:modal` pseudo class) landed in a spec yet? I know it was [resolved to be added](https://github.com/w3c/csswg-drafts/issues/6965#issuecomment-1118033655), but I hadn't seen it land. And this change causes all browsers to fail this test now. (Also, this test is part of Interop2022, so it kind of sucks to change it like this now.)

Reply:
Found the spec: https://drafts.csswg.org/selectors/#modal-state

Maybe ok to keep this test, I suppose. Still feels odd to add things to Interop2022 mid-year that were actually added to specs mid-year. But ok, I suppose this is a simple-enough addition.

Reply:
@mfreed7 can you weigh in on https://github.com/web-platform-tests/interop-2022/issues/79?

Reply:
Note that we generally want the tests to stay the same throughout the year, but we don't have anything in place to prevent test updates. Instead we rely on review after the fact if it happens.